validphys2/src/validphys/photon/compute.py

```diff
@@ -173,7 +175,7 @@ def compute_photon_array(self, xgrid, id):
         # we want x * gamma(x)
         return xgrid * photon_Q0
     
-    def produce_interpolator(self):
+    def produce_interpolators(self):
         # TODO : pass the grid in a more clever way
```

Comment:
A more clever way might be:
- either make `vp` depend on `eko`, and you use `lambertgrid` (best option)
- or you dump on a file and load with YAML